src/pages/payment-methods/_expanded-list.tsx

```diff
@@ -149,7 +149,11 @@ const ExpandList = ({ payment_data, is_fiat_onramp, locale }: PaymentProps) => {
                 {!is_fiat_onramp && (
                     <Td>
                         <>
-                            {Array.isArray(payment_data.min_max_withdrawal) ? (
+                            {payment_data?.minimum_withdrawal ? (
```

Comment:
IMO it's better we handle it with `is_loading` and the `error` handling cause at first before we get the response from API we are showing mock data values which I think it will be misleading for the client . 🙏